gtsam/inference/Ordering.h

```diff
@@ -58,13 +61,15 @@ class Ordering: public KeyVector {
       Base(keys.begin(), keys.end()) {
   }
 
+#ifdef GTSAM_USE_BOOST_FEATURES
   /// Add new variables to the ordering as ordering += key1, key2, ...  Equivalent to calling
   /// push_back.
   boost::assign::list_inserter<boost::assign_detail::call_push_back<This> > operator+=(
       Key key) {
     return boost::assign::make_list_inserter(
         boost::assign_detail::call_push_back<This>(*this))(key);
   }
```

Comment:
We could take the stance that, if you want to remove BOOST, this is what goes as well. By default, this will still work... But I can see that down the line we might want to make the flag OFF by default - in which case they will see errors. Not sure what the right move is yet :-(

Reply:
Oh true I wasn't thinking straight and forgot this only breaks if set BOOST OFF, even though that's literally the change 😂 nvm!